packages/webpack-extraction-plugin/src/webpackLoader.test.ts

```diff
@@ -25,6 +25,9 @@ async function compileSourceWithWebpack(entryPath: string, options: CompileOptio
       filename: 'bundle.js',
       pathinfo: false,
     },
+    externals: {
+      '@griffel/react': 'Griffel',
+    },
```

Comment:
Using `externals` is better:
- we don't need Griffel itself for these tests
- with `externals` we don't depend on build artifacts from other packages